.github/workflows/release-plugins.yaml

```diff
@@ -0,0 +1,114 @@
+name: Backstage knative-event-mesh-backend
+
+on:
+  pull_request:
+    branches:
+      - 'release-*'
+    paths:
+      - 'backstage/plugins/knative-event-mesh-backend/**'
+  push:
+    branches:
+      - 'release-*'
+    paths:
+      - 'backstage/plugins/knative-event-mesh-backend/**'
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup release name for release branch push
+        if: github.event_name == 'push'
+        run: |
+          BRANCH=$(git rev-parse --abbrev-ref HEAD) #e.g. release-v1.33
+          echo "Release branch: ${BRANCH}"
+          GO111MODULE=off go get -u github.com/openshift-knative/hack/cmd/sobranch
+          SO_RELEASE_NAME=$( $(go env GOPATH)/bin/sobranch --upstream-version "${BRANCH}") # input format: "release-v1.11" or "release-1.11" or "v1.11" or "1.11"
```

Comment:
I think we need to install Go before this step?

Reply:
Well, it works here: https://github.com/aliok/backstage-plugins-no-fork-test/actions/runs/8999257248

Reply:
We need to make sure that the version is what we need for building the tool

Reply:
I didn't understand, sorry. Go must be there, otherwise it would've failed on this action that I've run on a fork: https://github.com/aliok/backstage-plugins-no-fork-test/actions/runs/8999257248/job/24721391861#step:3:26

Reply:
Yes, my point is it is probably using some default installation and default version will change without notice (since we use "ubuntu-latest"), I think we need to make sure we're installing the correct Go version and the Go version we use in the workflow is version controlled, otherwise the build might break and we won't notice

Reply:
https://github.com/openshift-knative/hack/blob/8ac18ac0a22c768fea94beaf41694e42ea54fe27/.github/workflows/go-unit-tests.yaml#L17-L20